### ISSUE TITLE
Can't ctrl+click ctor members in initializer list to get to declaration in the header

### ISSUE THREAD

**Issue by azydevelopment** [microsoft/vscode-cpptools#1087]
- VSCode Version: Code 1.16.1 (27492b6bf3acb0775d82d2f87b25a93490673c6d, 2017-09-14T16:38:23.027Z)
- OS Version: Windows_NT x64 10.0.15063
- Extensions: 

Extension|Author (truncated)|Version
---|---|---
cpptools|ms-|0.13.0

---

Steps to Reproduce:

1. Open attached test project
[test.zip](https://github.com/Microsoft/vscode-cpptools/files/1347486/test.zip)
2. Open foo.cpp
3. Ctrl+click the member initialization of the 'bar' member

Expected: Goes to the declaration of the member in the header foo.h
Actual: Nothing

Reproduces without extensions: Yes/No

**Comment by bobbrow** [comment]
inlining the issue:

```
class Foo {
    Foo();
    bool bar;
};

Foo::Foo() : bar(false) { // <-- VSCode can't ctrl+click 'bar' to get to the declaration above
    bar = false;          // <-- ctrl+click on 'bar' here works.
}
```

**Comment by azydevelopment** [comment]
Wondering if this is on the radar anywhere. Also just putting a comment here so it doesn't autoclose :P.

**Comment by bobbrow** [comment]
It might be covered by #255 when we change from using the browse engine for code navigation to using the semantic information from the IntelliSense engine.  This works in Visual Studio, so I suspect it will work for us when we get that code ported over.

**Comment by azydevelopment** [comment]
Looks like the bug this is duped to is marked 'on deck'. Does it have an ETA?

**Comment by sean-mcmanus** [comment]
We plan to get started on the work soon -- ETA 3-6 months? We're not really sure.

**Comment by sean-mcmanus** [comment]
@azydevelopment This has been fixed with 0.19.0 -- so my ETA of 3-6 months was way off, although we still have work to do if the definition is not in the current translation unit.

**Comment by bobbrow** [comment]
Since this issue is about declarations (and the example above works now), I will close it.
